app/controllers/v2/systems_controller.rb

```diff
@@ -94,6 +96,13 @@ def build_tailorings!
       end
     end
 
+    def systems_with_sibling_policies
+      V2::PolicySystem.joins(policy: :profile)
+                      .where(profile: { ref_id: policy.ref_id }, system_id: permitted_params[:ids])
+                      .where.not(policy_id: policy.id)
+                      .select(:system_id)
```

Comment:
Technically this can return more than it is within in pundit scope if the right combination of ref_id and system_id is provided. At this use-case it is not an issue, but I'd harden it nevertheless (unless it is already).

Reply:
Nice catch, but for that I would have to join with the systems endpoint and that makes the querying even more expensive. This is only being used in an already scoped-down environment, so would you be okay with just adding a comment above it?